mlx_graphs/nn/message_passing.py

```diff
@@ -11,7 +13,7 @@ class MessagePassing(nn.Module):
     passing paradigm. This implementation is inspired from PyTorch Geometric [2].
 
     Args:
-        aggr (str): the aggregation strategy used to aggregate messages
+        aggr (str): Aggregation strategy used to aggregate messages
```

Comment:
Please add the available options.

Also, should `aggr` be a `Literal` with the different available options rather than a string? Even an `Enum` but that may add some extra burden to the UX.